src/restconfig-ng/src/main/java/org/geoserver/catalog/rest/CatalogController.java

```diff
@@ -36,7 +36,13 @@
     /**
      * Not an official MIME type, but GeoServer used to support it
      */
+    public static final MediaType MEDIATYPE_TEXT_JSON = MediaType.valueOf(CatalogController.TEXT_JSON);
```

Comment:
We should probably actually move all of these into their own class, and match the MediaType naming scheme, e.g:
```
TEXT_JSON
TEXT_JSON_VALUE
TEXT_XML
TEXT_XML_VALUE
```